src/index.js

```diff
@@ -0,0 +1,142 @@
+const state = {
+  tempCounter: 72,
+  lat: '49.7818607',
+  lon: '20.9440505',
+  city: 'Kąśna Dolna, Poland'
+};
+
+const increaseTemp = (event) => {
+  state.tempCounter += 1;
+  const tempCounterContainer = document.getElementById('tempNumber');
+  tempCounterContainer.textContent = state.tempCounter + '°';
+  let tempCircle = document.getElementById('circle');
+  tempCircle = displayColorAndGarden(event);
+};
+
+const decreaseTemp = (event) => {
+  state.tempCounter -= 1;
+  const tempCounterContainer = document.querySelector('#tempNumber');
+  tempCounterContainer.textContent = state.tempCounter +'°';
+  let tempCircle = document.getElementById('circle');
+  tempCircle = displayColorAndGarden(event);
+};
+
+const displayColorAndGarden = (event) => {
+  let tempCircleColor = document.getElementById('tempNumber');
+  let gardenContainer = document.getElementById('earthGarden')
+  if (state.tempCounter >= 80) {
+    tempCircleColor.style.color = 'red';
+    gardenContainer.textContent = '🌵__🐍_🦂_🌵🌵__🐍_🏜_🦂';
+  } else if (state.tempCounter <= 79 && state.tempCounter >= 70) {
+    tempCircleColor.style.color = 'orange';
+    gardenContainer.textContent = '🌸🌿🌼__🌷🌻🌿_☘️🌱_🌻🌷';
+  } else if (state.tempCounter <= 69 && state.tempCounter >= 60) {
+    tempCircleColor.style.color = 'yellow';
+    gardenContainer.textContent = '🌾🌾_🍃_🪨__🛤_🌾🌾🌾_🍃';
+  } else if (state.tempCounter <= 59 && state.tempCounter >= 50) {
+    tempCircleColor.style.color = 'aquamarine';
+    gardenContainer.textContent = '🌲🌲⛄️🌲⛄️🍂🌲🍁🌲🌲⛄️🍂🌲';
+  } else if (state.tempCounter <= 49 && state.tempCounter >= 30) {
+    tempCircleColor.style.color = 'teal';
+    gardenContainer.textContent = '🌲🌲⛄️🌲⛄️🍂🌲🍁🌲🌲⛄️🍂🌲';
+  }
+};
+
+const displaySky = (event) => {
+  let newOption = event.target.value;
+  let changeSkyEl = document.getElementById('skyGarden');
+  if (newOption === 'sunny') {
+    changeSkyEl.textContent = '☁️ ☁️ ☁️ ☀️ ☁️ ☁️'
+  } else if (newOption === 'cloudy') {
+    changeSkyEl.textContent = '☁️☁️ ☁️ ☁️☁️ 🌤 ☁️ ☁️☁️'
+  } else if (newOption === 'rainy') {
+    changeSkyEl.textContent = '🌧🌈⛈🌧🌧💧⛈🌧🌦🌧💧🌧🌧';
+  } else if (newOption === 'snowy') {
+    changeSkyEl.textContent = '🌨❄️🌨🌨❄️❄️🌨❄️🌨❄️❄️🌨🌨'
+  }
+};
+
+function changeCity(e) {
+  const changeHeading = document.getElementById('miniheader');
+  changeHeading.textContent = e.target.value;
+  state.city = changeHeading.textContent;
+}
+
+function resetCity() {
+  const cityInput = document.querySelector('input');
+  cityInput.value = 'Kąśna Dolna, Poland';
```

Comment:
Consider creating a constant like `DEFAULT_CITY` for 'Kąśna Dolna, Poland' instead of using a string literal here. It's good practice to use constants when the literal value won't change and also if someone reads 
```javascript 
cityInput.value = DEFAULT_CITY;
```

the intent is a little more clear.